[FW][FIX] l10n_ar_account_withholding: astericos al ver clave CIT ARBA

fw-bot-adhoc opened the pull request:
Ticket: 87746

Ponemos astericos al ver clave CIT ARBA al ir a "Contabilidad / Configuración / Ajustes".

Sin este pr:
![image](https://github.com/user-attachments/assets/f29fec41-6eb3-45d3-8475-61cd9f29accb)

Con el cambio introducido en este pr:
![image](https://github.com/user-attachments/assets/8ed0e024-48d5-4ccc-84c8-083a1ff307cc)

Forward-Port-Of: ingadhoc/odoo-argentina#1051

pablohmontenegro committed:
[FIX] l10n_ar_account_withholding: astericos al ver clave CIT ARBA

Ticket: 87746
X-original-commit: 8f8c67e8bb2ca887d11405931e88554a0de85a70

roboadhoc commented:
[Pull request status dashboard](https://mergebot.adhoc.ar/ingadhoc/odoo-argentina/pull/1060).

fw-bot-adhoc commented:
@pablohmontenegro @jue-adhoc cherrypicking of pull request ingadhoc/odoo-argentina#1051 failed.

stdout:
```
CONFLICT (modify/delete): l10n_ar_account_withholding/wizard/res_config_settings_views.xml deleted in HEAD and modified in 6fe47929 ([FIX] l10n_ar_account_withholding: astericos al ver clave CIT ARBA). Version 6fe47929 ([FIX] l10n_ar_account_withholding: astericos al ver clave CIT ARBA) of l10n_ar_account_withholding/wizard/res_config_settings_views.xml left in tree.
CONFLICT (modify/delete): l10n_ar_account_withholding/__manifest__.py deleted in HEAD and modified in 6fe47929 ([FIX] l10n_ar_account_withholding: astericos al ver clave CIT ARBA). Version 6fe47929 ([FIX] l10n_ar_account_withholding: astericos al ver clave CIT ARBA) of l10n_ar_account_withholding/__manifest__.py left in tree.

```

stderr:
```
14:31:44.059292 git.c:444               trace: built-in: git cherry-pick 6fe47929150c23acccc4a2107c33b910699ff24e
error: could not apply 6fe47929... [FIX] l10n_ar_account_withholding: astericos al ver clave CIT ARBA
hint: after resolving the conflicts, mark the corrected paths
hint: with 'git add <paths>' or 'git rm <paths>'
hint: and commit the result with 'git commit'
----------
status:

```

Either perform the forward-port manually (and push to this branch, proceeding as usual) or close this PR (maybe?).

In the former case, you may want to edit this PR message as well.

More info at https://github.com/odoo/odoo/wiki/Mergebot#forward-port
